.github/workflows/github-actions.yml

```diff
@@ -7,7 +7,7 @@ on:
             - '*'
 jobs:
     unit-test:
-        container: cuaesd/aesd-autotest:24-unit-test
+        container: docker.io/cuaesd/aesd-autotest:24-unit-test
```

Comment:
@JLDenn looks like you made this change because you are using podman instead of the Docker?  I haven't tested on podman yet and this particular change is likely to cause merge conflicts with future assignments.  If possible to get a setup matching the course requirements this may help alleviate future issues.  If you are willing to work around issues and merge conflicts it's fine to use an alternate setup.